### PR TITLE
Refactor TTL flags

### PR DESCRIPTION
**Pull request by miguelbernadi**

### CODE REVIEW

File: mcc/ttl/main.go [hunk 1]
```diff
@@ -255,7 +257,7 @@ func main() {
 	if len(list2) > 0 {
 		list = list2
 	}
-	changeResponse, err := UpsertResourceRecordSetTTL(list, *ttl, zoneID, svc)
+	changeResponse, err := UpsertResourceRecordSetTTL(list, ttl, zoneID, svc)
```
Comment (ifosch) [inline mcc/ttl/main.go:260]: Being `ttl` a global variable, do we still need to pass it?
Maybe doing `svc` a global variable could also help... but I'm guessing some of the following PRs are doing other things related to these.

**Comment by ifosch** [comment]
+1

**Comment by miguelbernadi** [comment]
svc as a global is sensible in my opinion, but ttl is not as we may want to
use that function several times with different ttl values. E.g. create
changesets for all entries in a zone with different ttls for some of them,
making the whole action atomic.

**Comment by ifosch** [comment]
Then I think `ttl` should not be a global, since different concurrent instances of the function could get it wrong, couldn't it?

**Pull request merged**